sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -46,6 +47,18 @@ def test_error_messages_on_wrong_input():
                              [0, 1, 0], [[1, 1], [0, 0]])
 
 
+def test_generalized_average():
+    a, b = 1, 2
+    methods = ["min", "sqrt", "sum", "max"]
+    means = [_generalized_average(a, b, method) for method in methods]
+    assert means[0] <= means[1] <= means[2] <= means[3]
+    c, d = 12, 12
+    means = [_generalized_average(c, d, method) for method in methods]
+    assert_equal(means[0], means[1])
+    assert_equal(means[1], means[2])
```

Comment:
We no longer use nosetests. Use bare assert instead of such helper functions in all new code